tools/k8s/gateways/contour-provisioner.yaml

```diff
@@ -11419,7 +11419,7 @@ spec:
                   - name
                   - supportedKinds
                   type: object
-                maxItems: 64
```

Comment:
I don't think we need to bump all of these but it's not just this path
```    # cur_value=$(yq '.spec.versions[0].schema.openAPIV3Schema.properties.spec.properties.listeners.maxItems' $filename)```

and I'm not sure which other one is key here. needs more time

Reply:
```
yq e 'select(.metadata.name == "gateways.gateway.networking.k8s.io")' tools/k8s/gateways/contour-provisioner.yaml | yq '.spec.versions[].schema.openAPIV3Schema.properties.spec.properties.listeners.maxItems'
```
only these two probably? testing it